aries-site/src/examples/cardPreviews/header.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Box, Header, Button, Text } from 'grommet';
+import { Hpe, Menu } from 'grommet-icons';
+
+export const HeaderPreview = () => {
+  return (
+    <Header background="background-front" fill="horizontal">
```

Comment:
Can we add a bottom border here to match the designs a little closer? ignore the blue strip on the side, that was just in the screen shot
<img width="344" alt="Screen Shot 2020-04-07 at 6 17 59 PM" src="https://user-images.githubusercontent.com/12522275/78734078-23723380-78fc-11ea-83e9-35b48117cf74.png">

Reply:
your eyes are better then Mine yes I will add that I wonder if there was also one on the footer I can add to make it feel separate